commcare_connect/templates/opportunity/opportunity_list.html

```diff
@@ -3,6 +3,63 @@
 
 {% block title %}{{ request.org }} - Opportunities{% endblock %}
 
+{% block inline_javascript %}
+  {{ block.super }}
+  <script>
```

Comment:
Is there a way to make this more reusable, since we will likely need sortable columns in other tables